.eslintrc

```diff
@@ -253,6 +256,91 @@
     "space-infix-ops": 2, // http://eslint.org/docs/rules/space-infix-ops
     "space-in-parens": [2, "never"], // http://eslint.org/docs/rules/space-in-parens
     "spaced-comment": [2, "always"], // http://eslint.org/docs/rules/spaced-comment
-    "arrow-spacing": [2, { "before": true, "after": true }] // https://eslint.org/docs/rules/arrow-spacing
+    "arrow-spacing": [2, { "before": true, "after": true }], // https://eslint.org/docs/rules/arrow-spacing
+
+    /**
+    * Additional rules to enable one by one
+    */
+    // "array-bracket-spacing": ["error", "never"],
+    // "array-callback-return": ["error", { "allowImplicit": true }],
+    // "arrow-body-style": ["error", "as-needed", { "requireReturnForObjectLiteral": false }],
+    // "arrow-parens": [ "error", "as-needed", { "requireForBlockBody": true }],
```

Comment:
This may need to be "always" here based on our prior rules.

Reply:
I'm cool with that